tests/jballerina-unit-test/src/test/resources/test-src/expressions/invocations/function-stmt.bal

```diff
@@ -60,3 +78,16 @@ function addFloat5 (float x, float y) returns (float) {
     float z = x + y;
     return z;
 }
+
+function testRecursiveFunctionWhichYields() {
+   bar(0);
+}
+
+function bar(int x) {
```

Comment:
Shall we add a test for stackoverflow. we can use same test with x == 10000 something

Reply:
Will look into this.